app-ios/tutanota/Info.plist

```diff
@@ -27,6 +27,7 @@
 			<string>de.tutao.tutanota</string>
 			<key>CFBundleURLSchemes</key>
 			<array>
+				<string>tutashare</string>
```

Comment:
nah, and as a corollary, we also can't prevent other apps to use the same scheme.
It would be possible to move the info about the required information to the host part of the url,

Reply:
Hmmm, I guess since we try to app group storage no data will be leaked anyway, right?